src/View/Helper/UrlHelper.php

```diff
@@ -52,8 +52,10 @@ public function build($url = null, $options = false)
         }
         $options += $defaults;
 
+        /** @var string $url */
         $url = Router::url($url, $options['fullBase']);
         if ($options['escape']) {
+            /** @var string $url */
```

Comment:
This seems quite redundant and unnecessary  - but I guess that means that phpstan needs it.

Reply:
It's need because return type for `h()` is `string|array`.